src/run.rs

```diff
@@ -0,0 +1,48 @@
+use chrono::prelude::*;
+use clap::ArgMatches;
+
+use crate::util::file::*;
+
+fn get_day(matches: &ArgMatches) -> u32
+{
+    if let Some(day) = matches.get_one::<String>("day")
+    {
+        day.parse().unwrap()
+    }
+    else
+    {
+        Utc::now().day()
+    }
+}
+
+pub async fn run(matches: &ArgMatches) -> Result<(), Box<dyn std::error::Error>>
+{
+    let day = get_day(matches);
+
```

Comment:
Should we maybe give a reasonable error message here if the day is not between 1 and 25? :thinking: 
For example:
```rs
assert!(day < 26 && day > 0, "Not possible to run aoc for day {}", day);
```

Just a thought :man_shrugging:

Reply:
👍